integrationtests/gitjob/controller/suite_test.go

```diff
@@ -97,6 +98,7 @@ var _ = BeforeSuite(func() {
 		Scheduler:  sched,
 		GitFetcher: fetcherMock,
 		Clock:      reconciler.RealClock{},
+		Recorder:   mgr.GetEventRecorderFor(fmt.Sprintf("gitjob-controller%s", "")),
```

Comment:
nit: this could simply be:
```suggestion
		Recorder:   mgr.GetEventRecorderFor("gitjob-controller"),
```